### ISSUE TITLE
Create new BlindSensor class.

### ISSUE THREAD

**Issue by kingsleyadam** [kingsleyadam/local-abbfreeathome#82]
Split from #74 , we need a new class `CoverSensor` that can be used to track the outputs of a Cover device. Fyi @nrenner73

Functions to include.

```python
FID_BLIND_SENSOR = 3
FID_BLIND_SENSOR_ROCKER_TYPE0 = 4160
FID_BLIND_SENSOR_ROCKER_TYPE1 = 4161
FID_BLIND_SENSOR_ROCKER_TYPE2 = 4162
FID_BLIND_SENSOR_PUSHBUTTON_TYPE0 = 4168
FID_BLIND_SENSOR_PUSHBUTTON_TYPE1 = 4169
FID_BLIND_SENSOR_PUSHBUTTON_TYPE2 = 4170
FID_BLIND_SENSOR_PUSHBUTTON_TYPE3 = 4171
```

```
"ABB7F6312A91": {
        "deviceReboots": "20",
        "floor": "02",
        "room": "14",
        "interface": "RF",
        "deviceId": "2039",
        "displayName": "Jalousien zentral",
        "unresponsive": false,
        "unresponsiveCounter": 0,
        "defect": false,
        "channels": {
          "ch0000": {
            "displayName": "Jalousien zentral",
            "floor": "02",
            "function": "FID_BLIND_SENSOR",
            "functionID": "3",
            "inputs": {
              "idp0001": {
                "pairing": "AL_NIGHT",
                "pairingID": 18,
                "value": "1"
              },
              "idp0002": {
                "pairing": "AL_INFO_ERROR",
                "pairingID": 273,
                "value": "0"
              },
              "idp0005": {
                "pairing": "AL_SYSAP_INFO_ERROR",
                "pairingID": 278,
                "value": "0"
              }
            },
            "outputs": {
              "odp0002": {
                "pairing": "AL_MOVE_UP_DOWN",
                "pairingID": 32,
                "value": "1"
              },
              "odp0003": {
                "pairing": "AL_STOP_STEP_UP_DOWN",
                "pairingID": 33,
                "value": "1"
              },
              "odp0006": {
                "pairing": "AL_SCENE_CONTROL",
                "pairingID": 4,
                "value": "0"
              }
            },
            "parameters": {
              "par0002": "30",
              "par0001": "50",
              "par0007": "1"
            },
            "room": "14"
          }
        },
        "parameters": {
          "par00ed": "1"
        }
      },
```

**Comment by derjoerg** [comment]
I have at least devices with the function `FID_BLIND_SENSOR`. I will look at them.

**Comment by derjoerg** [comment]
> I have at least devices with the function `FID_BLIND_SENSOR`. I will look at them.

Just as an FYI: I just made a test and it doesn't matter if I assign a rocker to a blind or e.g. an awning, the function will always stay with `FID_BLIND_SENSOR`. So I would stick to the name `BlindSensor` for this device and not `CoverSensor`.

And all the functions listed above also start with `FID_BLIND_SENSOR_xxx`

OK?

**Comment by derjoerg** [comment]
see #88

**Comment by kingsleyadam** [comment]
> see #88

Just a heads up, if you use the term closes it'll link the PR to the issue and close it automatically when PR is merged.

closes #88
